contracts/BondingClaimer.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import {IBondingClaimer} from "./interfaces/IBondingClaimer.sol";
+import {IBondStorage} from "./interfaces/IBondStorage.sol";
+
+contract BondingClaimer is IBondingClaimer {
+
+    constructor(IBondStorage _bondStorage) {
+        bondStorage = _bondStorage;
+    }
+
+    /* ========== CONSTANTS ========== */
```

Comment:
We don't need empty sections, if we don't need ex. constants here - better to remove the section altogether

Reply:
For sure, I left it to keep everything on the place straight forward

> @GooGrand resolving for now but we need to tidy it up before release